spec/models/comment_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  before do
+    @post = create(:post, message: "Hello")
+    @comment = Comment.new(message: "generic comment", post_id: @post.id)
```

Comment:
get factory girl to do this:
Look up traits for a factory: https://robots.thoughtbot.com/never-fear-traits-are-here

```ruby
create(:post, :with_comment)
```